### PR TITLE
Twig preview (4.12)

### PR DESCRIPTION
**Pull request by m-vo**

### CODE REVIEW

File: docs/dev/framework/templates/twig.md [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+without the need to know about the others because every *extend* uses the
+managed `@Contao` namespace:
+
+```twig
+{% import '@ContaoCore/Image/Studio/_macros.html.twig' as studio %}
+
+<section class="card">
+  <div class="inner">
+    <header class="title">
+      <div class="inner">🤩 {{ title }} 🤯</div>
```
Comment (ausi) [inline docs/dev/framework/templates/twig.md]: My brain-based parser outputs this the other way round.
Is it wrong in the documentation, the implementation or my brain? ☺️
```suggestion
      🤩 <div class="inner">{{ title }}</div> 🤯
```

Comment (m-vo) [reply docs/dev/framework/templates/twig.md]: Eh, of course :smile:.  Must have been me using brain-based parsing when writing.

File: docs/dev/framework/templates/twig.md [hunk 2]
```diff
@@ -290,24 +291,41 @@ template.
...
 
 {{< tabs groupId="twig">}}
 {{% tab name="PHP" %}}
 ```html
 <?php echo $this->normalValue; ?>
-<?php echo $this->lazyValue1(); ?>
-<?php echo $this->lazyValue2('foo'); ?>
+<?php echo $this->lazyValue(); ?>
+<?php echo $this->fooFunction('bar'); ?>
 <?php echo implode(', ', $this->lazyArray()); ?>
```
Comment (ausi) [inline docs/dev/framework/templates/twig.md]:
```suggestion
<?php echo $this->lazyValue; ?>
<?php echo $this->fooFunction('bar'); ?>
<?php echo implode(', ', $this->lazyArray); ?>
```
You don’t need the braces in the legacy templates.

Comment (m-vo) [reply docs/dev/framework/templates/twig.md]: Thanks, wasn't sure about this anymore!

fixed in 289627d

**Review by ausi — APPROVED** [review]

**Review by fritzmg — CHANGES_REQUESTED** [review]
Can you change all `<?php echo …; ?>` instances to `<?= … ?>`? That's the standard within the core :)

**Comment by m-vo** [comment]
> Can you change all <?php echo …; ?> instances to <?= … ?>? That's the standard within the core :)

Regexed in b60ddba :slightly_smiling_face:

**Review by fritzmg — CHANGES_REQUESTED** [review]
The indentations of the examples are not uniform. In the Contao core the PHP templates use 2 spaces for indentations while twig templates use 4 spaces. Should we use 2 or 4 spaces for all examples (php and twig)? 🤔 To visualise the comparisons better I would use the same indentation for both here.

**Comment by m-vo** [comment]
> To make visualise the comparisons better I would use the same indentation for both here.

Agreed. I normalized everything to 2 spaces in fae97be.

**Review by fritzmg — APPROVED** [review]

**Comment by fritzmg** [comment]
Thank you @m-vo :)

**Pull request merged**